com.civicrm.hrjobroles/js/src/job-roles/controllers/job-roles.controller.js

```diff
@@ -475,6 +476,19 @@ define([
       });
     }
 
+    /**
+     * Trigger opening of editor window for editing provided option type
+     *
+     * @param url
```

Comment:
I don't think you need URL to pass considering which URLs I see you pass in HTML templates:

```js
function openOptionsEditor (optionType) {
  var url = '/civicrm/admin/options/' +optionType + '?reset=1';

  crmAngService.loadForm(url)
    .on('crmUnload', function () {
      getOptionGroupTypeValues(optionType);
    });
}
```